### PR TITLE
Fix imports for abc-init

### PR DESCRIPTION
**Pull request by jedbrown**
These were introduced when these two commits missed updating __main__.py
8a9ab65cce652cf22c3785e6bde9809c23d45f90
e99472705a3fc9dcd2c80e95b5be58d02d4a9898

Cc: @kcranston (thanks for your work; just noticed these issues)

### CODE REVIEW

File: abcclassroom/__main__.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 from . import template
 from . import feedback as fdback
 from . import github
+from . import git
```
Comment (kcranston) [inline abcclassroom/__main__.py:8]: I think what we want to do here is also import auth in addition to git and github

File: abcclassroom/__main__.py [hunk 2]
```diff
@@ -37,7 +38,7 @@ def init():
     """
 
     print("Step 1: Setting up GitHub API access")
-    github.get_access_token()
+    github.auth.get_access_token()
```
Comment (kcranston) [inline abcclassroom/__main__.py:41]: and then this is just auth.get_access_token()

File: abcclassroom/github.py [hunk 3]
```diff
@@ -9,6 +9,8 @@
 
 import github3 as gh3
 
+from . import auth
```
Comment (kcranston) [inline abcclassroom/github.py:12]: and we don't need to import auth here

**Comment by kcranston** [comment]
Thanks for catching the missing imports! Left a few comments to avoid accessing the auth methods through the github methods. Let me know what you think.

**Comment by jedbrown** [comment]
Thanks; revised.

**Pull request merged**